openwhisk/README.md

```diff
@@ -1,262 +1,569 @@
-# OpenWhisk Runtimes for Knative
+# OpenWhisk Application pipeline for Knative
 
-This directory contains Tekton Task which can be used to Build and Serve Knative compatible applications (i.e., OpenWhisk Actions) on Kubernetes.
+*This catalog offering provides a single pipeline that can be used to build either [Apache OpenWhisk](https://openwhisk.apache.org/) or [Knative](https://openwhisk.apache.org/) compatible containers for supported runtimes used to execute OpenWhisk serverless functions.*
```

Comment:
please update the knative link here

Reply:
Thanks @pritidesai.  Silly copy/paste mistake.  Will take the opportunity to add links a couple of other places in the opening paragraphs that may be helpful for easily accessing context.